### ISSUE TITLE
We are not able to find coins matching your keywords

### ISSUE THREAD

**Issue by bdmorin** [bichenkk/coinmon#15]
Just loaded fresh via npm, -f works, but no other options.
```
-> % coinmon -t 50              
                                                            
  ___     ___    __     ___      ___ ___      ___     ___    
 /'___\  / __`\ /\_\  /' _ `\  /' __` __`\   / __`\ /' _ `\  
/\ \__/ /\ \_\ \\/\ \ /\ \/\ \ /\ \/\ \/\ \ /\ \_\ \/\ \/\ \ 
\ \____\\ \____/ \ \ \\ \_\ \_\\ \_\ \_\ \_\\ \____/\ \_\ \_\
 \/____/ \/___/   \/_/ \/_/\/_/ \/_/\/_/\/_/ \/___/  \/_/\/_/
                                                             


We are not able to find coins matching your keywords
-> % coinmon
                                                            
  ___     ___    __     ___      ___ ___      ___     ___    
 /'___\  / __`\ /\_\  /' _ `\  /' __` __`\   / __`\ /' _ `\  
/\ \__/ /\ \_\ \\/\ \ /\ \/\ \ /\ \/\ \/\ \ /\ \_\ \/\ \/\ \ 
\ \____\\ \____/ \ \ \\ \_\ \_\\ \_\ \_\ \_\\ \____/\ \_\ \_\
 \/____/ \/___/   \/_/ \/_/\/_/ \/_/\/_/\/_/ \/___/  \/_/\/_/
                                                             


We are not able to find coins matching your keywords
```
I'll try to figure out what's going on, thought you'd like to know.

**Comment by maticrivo** [comment]
I fixed it in my PR #16

**Comment by bichenkk** [comment]
Thanks for reporting. I have just released version 0.0.9.
Please do
```
npm update -g coinmon
```
